### ISSUE TITLE
Not stable result on kitti dataset in Cyclist and Pedestrain category in validation dataset.

### ISSUE THREAD

**Issue by kaixinbear** [TRAILab/CaDDN#92]
Hi~
I notice that tha paper shows the result on pedestrain and cyclist AP only in kitti test set not on validation set.
And I find the huge gap between these two classes on these two set.
Below the result trained on ResNet50 with batchsize 2
![1637984777(1)](https://user-images.githubusercontent.com/33575107/143667049-fc95aed7-5102-4cb5-9d7c-3ae7d0b75ab8.png)
While your paper gives the result as follows:
![1637984830(1)](https://user-images.githubusercontent.com/33575107/143667061-2df46ef4-6fbd-4707-af32-6d624364fa1c.png)
which is much higher than the result on validation.
Could you please tell me why this phenomenon occurs? And could you please offer me the result on Cyclist and Pedestrain category in validation dataset  ?
